estimate-web/src/components/UserStoryCommentEditor.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from "react";
+import { PropTypes } from "prop-types";
+
+import {
+    TextField, Persona, PersonaSize, PrimaryButton, Spinner
+} from "office-ui-fabric-react";
+
+import "./UserStoryCommentEditor.less";
+import { connect } from "react-redux";
+import userShape from "../reducers/models/userShape";
+import { requestWorkItemAddComment } from "../actions/devops";
+import workItemShape from "../reducers/models/workItemShape";
+
+class UserStoryCommentEditor extends Component {
+    state = {
+        comment: "",
+        workItem: undefined
+    };
+
+    static getDerivedStateFromProps(nextProps, prevState) {
+        if (nextProps.workItem !== prevState.workItem) {
+            // Clear comment when workItem changes
+            return { comment: "", workItem: nextProps.workItem };
+        }
+        return null;
+    }
+
+    onChange = (e, comment) => {
+        this.setState({ comment });
+    };
+
+    onAddClicked = () => {
+        const { addComment, workItem } = this.props;
+        const { comment } = this.state;
+
+        addComment(workItem.id, comment);
+
+        this.setState({ comment: "" });
+    };
+
+    render() {
+        const { user, addingComment } = this.props;
+        const { comment } = this.state;
+        return (
+            <div className="user-story-comment-editor">
+                <Persona
```

Comment:
I think you could reuse `EstimatorPersona` here, which would give you the connection status as part of the badge :-)

Reply:
To be honest, I don't think we need to show the connection status next to a work item's comment. The status is irrelevant there, as it's not related to voting.